spec/models/tracked_revisions_spec.rb

```diff
@@ -5,22 +5,34 @@
 
 describe 'Course#tracked_revisions' do
   let(:course) { create(:course, start: '10-07-2019'.to_date, end: '10-08-2019'.to_date) }
+  let(:different_wiki_course) do
+    create(:course,
+           title: 'Science',
+           school: 'WinterSchool',
+           term: 'Winter 2019',
+           slug: 'WinterSchool/Science_(Winter_2019)',
+           start: '10-07-2019'.to_date, end: '10-08-2019'.to_date,
+           home_wiki_id: 3)
+  end
   let(:user) { create(:user, username: 'Hbultra') }
 
   before do
     course.students << user
+    different_wiki_course.students << user
   end
 
   it 'fetches the course stats for only tracked articles' do
     # Fetch the original data
     VCR.use_cassette 'course_upload_importer/Hbultra' do
       UpdateCourseStats.new(course)
+      UpdateCourseStats.new(different_wiki_course)
     end
 
     # Check if the data is cached
     old_character_sum = course.character_sum
     old_course = course.dup
     expect(course.revision_count).to eq(old_course.revision_count)
+    expect(different_wiki_course.revision_count).to eq(0)
```

Comment:
Does this test fail if you undo the change to `course.rb`?

Reply:
Yes it gives the following failure error
Expected :0
Got: 41

Reply:
![Screenshot from 2020-03-17 22-40-50](https://user-images.githubusercontent.com/37243661/76883717-96aafb80-68a2-11ea-94b5-d5e43e2cc8b3.png)
![Screenshot from 2020-03-17 22-41-04](https://user-images.githubusercontent.com/37243661/76883732-9c084600-68a2-11ea-9f2a-2ae53a9db4dd.png)